extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -81,7 +81,8 @@ export class PostgresOverviewPage extends DashboardPage {
 		const titleCSS = { ...cssStyles.title, 'margin-block-start': '2em', 'margin-block-end': '0' };
 		content.addItem(this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
 			value: loc.serviceEndpoints,
-			CSSStyles: titleCSS
+			CSSStyles: titleCSS,
+			ariaRole: 'heading'
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role the heading role should also have an associated `aria-level` attribute.

But in general it's preferred for these things to use actual h1, h2, etc. elements which automatically get the correct roles read out. 

@alanrenmsft I'm thinking maybe we want to make the text component have an optional `headingLevel` property (number) that when set results in the text component being made as a h# instead of a span. What do you think of that?

Reply:
sounds good to me.

Reply:
Alrighty, I'll work on getting that added and send it out for review here shortly. @nasc17 You can just leave this open until that happens